cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -120,3 +124,98 @@ def cmp_function(_, full_data):
         event_type=event_type,
         first_event=True
     )
+
+
+@pytest.fixture
+def targeted_refresh(merkyl_inspector):
+    """
+    Targeted refresh registration and search log fixture
+
+    This fixture waits for targeted refresh of some entity
+    to be registered. It does not register anything by itself,
+    so you have to register it in test function.
+    Then it searches content of evm.log to check if targeted
+    refresh has occurred and raises an error if not so.
+
+    Usage:
+
+        def test_something(targeted_refresh)
+                trigger_targeted_refresh() <- some function that will trigger
+                                              targeted refresh
+                with targeted_refresh.target_timeout():
+                    targeted_refresh.register_target(domain_id, 'domain')
+                    targeted_refresh.register_target(cont_vport, 'vport', comment='L3 cont')
+    """
+    merkyl_inspector.add_log('/var/www/miq/vmdb/log/evm.log')
+    targets = set()
+
+    @contextmanager
+    def target_timeout():
+        yield
+
+        def check_log():
+            for target in set(targets):
+                success = merkyl_inspector.search_log(
+                    target[0],
+                    '/var/www/miq/vmdb/log/evm.log'
+                )
+
+                if success:
+                    targets.remove(target)
+            return len(targets) == 0
+
+        try:
+            wait_for(check_log,
+                     delay=5,
+                     num_sec=180,
+                     handle_exception=True)
```

Comment:
Not sure, but shouldn't this be set to False? I understand we now suppress all exceptions that may occur in `check_log` which I think is not desired. Please just delete the argument, False is default I see.